fix(chat): audio and popup alerts

JoVictorNunes opened the pull request:
<!--
PLEASE READ THIS MESSAGE.

HOW TO WRITE A GOOD PULL REQUEST?

- Make it small.
- Do only one thing.
- Avoid re-formatting.
- Make sure the code builds and works.
- Write useful descriptions and titles.
- Address review comments in terms of additional commits.
- Do not amend/squash existing ones unless the PR is trivial.
- Read the contributing guide: https://docs.bigbluebutton.org/support/faq.html#bigbluebutton-development-process
- Sign and send the Contributor License Agreement: https://docs.bigbluebutton.org/support/faq.html#why-do-i-need-to-sign-a-contributor-license-agreement-to-contribute-source-code

-->

### What does this PR do?

<!-- A brief description of each change being made with this pull request. -->

Fixes both audio and popup alerts of chat.

### Closes Issue(s)
<!-- List here all the issues closed by this pull request. Use keyword `closes` before each issue number
Closes #123456
-->
Closes none

ramonlsouza reviewed (changes requested):
if chat alerts are enabled when a poll is published, 2 alerts appear

![poll-alert](https://github.com/bigbluebutton/bigbluebutton/assets/3728706/fba28c38-2cc7-4021-b304-42748017ed80)


new alerts will always replace the last one... is this expected?

https://github.com/bigbluebutton/bigbluebutton/assets/3728706/6ec40f8e-292a-40a4-9f72-0ef2d2a07e44

alerts appear again if presenter is changed:

https://github.com/bigbluebutton/bigbluebutton/assets/3728706/86e1588f-dca8-41cc-a3a3-b49b75655d08

JoVictorNunes commented:
> if chat alerts are enabled when a poll is published, 2 alerts appear

Fixed in https://github.com/bigbluebutton/bigbluebutton/pull/19675/commits/e1ed124f93469b1efac999d281839ef1940fba5d along with https://github.com/bigbluebutton/bigbluebutton/pull/19675/commits/e913b985e3f2330692f22e1019ce00aa40e6daf6.

> alerts appear again if presenter is changed

Fixed in https://github.com/bigbluebutton/bigbluebutton/pull/19675/commits/7e04467558bcd2729cec7a64aa848e69bc5dc01d.

JoVictorNunes commented:
> new alerts will always replace the last one... is this expected?

Yes, this is expected. But this may be a point of discussion to see which is the best behavior. IMHO always displaying the last message is the way, since displaying one toast for every message can flood the user's screen. Another way is using the pagination technique, but this can create large toasts as well.
